### ISSUE TITLE
Segfault for ctrl+c on a new instance

### ISSUE THREAD

**Issue by domenukk** [vanhauser-thc/AFLplusplus#260]
This happens at least in QEMU mode and non-instrumented mode:
When I hit ctrl+c before fuzzing starts (after `All set and ready to roll`), AFL dies with `SIGSEV` and destroys the console output on it's way. 
![image](https://user-images.githubusercontent.com/297744/76805563-69494980-67df-11ea-9db9-f92ca3089174.png)

**Comment by domenukk** [comment]
Looks like it tries (and fails) to paint the ui one last time
```strace
23:45:00 write(1</dev/pts/5<char 136:5>>, "x\17  last uniq hang : \33[0mnone se"..., 105) = 105
 > /usr/lib/libc-2.31.so(__write+0x17) [0xf0567]
 > /usr/lib/libc-2.31.so(_IO_file_write+0x2d) [0x8124d]
 > /usr/lib/libc-2.31.so(new_do_write+0x66) [0x805a6]
 > /usr/lib/libc-2.31.so(_IO_do_write+0x19) [0x822e9]
 > /usr/lib/libc-2.31.so(_IO_file_xsputn+0xe6) [0x818f6]
 > /usr/lib/libc-2.31.so(__vfprintf_internal+0x7b8) [0x6aa98]
 > /usr/lib/libc-2.31.so(_IO_printf+0xaf) [0x572cf]
 > /mnt/c/Users/DMaier/tmp/AFLplusplus/afl-fuzz(show_stats+0xa9b) [0xce4b]
 > /mnt/c/Users/DMaier/tmp/AFLplusplus/afl-fuzz(main+0x3db7) [0x75d7]
 > /usr/lib/libc-2.31.so(__libc_start_main+0xf3) [0x27023]
 > /mnt/c/Users/DMaier/tmp/AFLplusplus/afl-fuzz(_start+0x2e) [0x8bbe]
23:45:00 write(1</dev/pts/5<char 136:5>>, "tq\17\33[0;36m cycle progress \16\33[1;9"..., 111) = 111
 > /usr/lib/libc-2.31.so(__write+0x17) [0xf0567]
 > /usr/lib/libc-2.31.so(_IO_file_write+0x2d) [0x8124d]
 > /usr/lib/libc-2.31.so(new_do_write+0x66) [0x805a6]
 > /usr/lib/libc-2.31.so(_IO_do_write+0x19) [0x822e9]
 > /usr/lib/libc-2.31.so(_IO_file_overflow+0x103) [0x82753]
 > /usr/lib/libc-2.31.so(_IO_puts+0x17a) [0x772da]
 > /mnt/c/Users/DMaier/tmp/AFLplusplus/afl-fuzz(show_stats+0xaa7) [0xce57]
 > /mnt/c/Users/DMaier/tmp/AFLplusplus/afl-fuzz(main+0x3db7) [0x75d7]
 > /usr/lib/libc-2.31.so(__libc_start_main+0xf3) [0x27023]
 > /mnt/c/Users/DMaier/tmp/AFLplusplus/afl-fuzz(_start+0x2e) [0x8bbe]
23:45:00 --- SIGSEGV {si_signo=SIGSEGV, si_code=SEGV_MAPERR, si_addr=0x12} ---
 > /mnt/c/Users/DMaier/tmp/AFLplusplus/afl-fuzz(show_stats+0xad1) [0xce81]
 > /mnt/c/Users/DMaier/tmp/AFLplusplus/afl-fuzz(main+0x3db7) [0x75d7]
 > /usr/lib/libc-2.31.so(__libc_start_main+0xf3) [0x27023]
 > /mnt/c/Users/DMaier/tmp/AFLplusplus/afl-fuzz(_start+0x2e) [0x8bbe]
```

**Comment by domenukk** [comment]
Fixed with Commit a57896a - it was a sideproduct of the new extra rendering step at the end.
